django/core/serializers/python.py

```diff
@@ -91,6 +92,7 @@ def Deserializer(object_list, **options):
     """
     db = options.pop('using', DEFAULT_DB_ALIAS)
     ignore = options.pop('ignorenonexistent', False)
+    handle_forward_references = options.pop('handle_forward_references', False)
```

Comment:
Hi @timgraham -- I'm afraid I don't understand your point.  loaddata is in control of how it deserializes objects, and so with my changes it shouldn't ever need to roll back a transaction to deal with forward references.

I've added some documentation in dced428e1248bb73cc1262ea16952fbc45e2aa27.  What's your opinion on the current state of this work?